firebase-config/src/test/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigTest.java

```diff
@@ -1551,6 +1552,24 @@ public void realtimeStreamListen_andUnableToParseMessage() throws Exception {
     verify(mockInvalidMessageEventListener).onError(any(FirebaseRemoteConfigClientException.class));
   }
 
+  @Test
+  public void realtime_updatesBackoffMetadataWithProvidedRetryInterval() throws Exception {
+    ConfigRealtimeHttpClient configRealtimeHttpClientSpy = spy(configRealtimeHttpClient);
+    when(mockHttpURLConnection.getResponseCode()).thenReturn(200);
+    int expectedRetryInterval = 240;
+    when(mockHttpURLConnection.getInputStream())
+        .thenReturn(
+            new ByteArrayInputStream(
+                String.format(
+                        "{ \"latestTemplateVersionNumber\": 1, \"retryIntervalSeconds\": %d }",
+                        expectedRetryInterval)
+                    .getBytes(StandardCharsets.UTF_8)));
+    when(mockFetchHandler.getTemplateVersionNumber()).thenReturn(1L);
+    configAutoFetch.listenForNotifications();
+
+    verify(sharedPrefsClient, times(1)).setRealtimeBackoffEndTime(any());
```

Comment:
Is there any way of getting the actual backoff value that was used instead of `any()`? 

I'd like more to check that the backoff time is X + expectedRetryInterval, than testing whether the method to set it has been called... just to be extra sure